### PR TITLE
kinder

### PR DESCRIPTION
**Pull request by fabriziopandini**
after https://github.com/kubernetes/kubeadm/pull/1446 merged, this PR adds the actual kinder code

/kind feature
/priority important-longterm

/assign @timothysc 
/assign @neolit123 

/cc @kubernetes/sig-cluster-lifecycle-pr-reviews

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module k8s.io/kubeadm
+
+go 1.12
```
Comment (neolit123) [inline go.mod]: hm, i though that it is possible to have the module under the kinder folder
e.g. `module k8s.io/kubeadm/kinder` ?

Comment (neolit123) [reply go.mod]: oh, there is one already...
in which case do we need this one?

Comment (fabriziopandini) [reply go.mod]: it is automatically generated, so I think it is needed...

Comment (neolit123) [reply go.mod]: isn't it possible to run `go module` only under the `kinder` folder and not the `/` folder?
or is this required for go 1.12 to work like so?

Comment (fabriziopandini) [reply go.mod]: if I use go modules in the kinder folder, go automatically generated go.mod in this folder and in the parent folder. It is required/managed by go

Comment (neolit123) [reply go.mod]: ok

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        "Wrapf",
+        "kcfg",
+        "kcluster",
+        "kctx",
+        "logrus",
+        "nginx",
+        "nslookup",
+        "srcs"
+    ]
+}
```
Comment (ereslibre) [inline .vscode/settings.json]: I would tend to not add specific IDE settings if possible.

File: kinder/OWNERS [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# See the OWNERS file documentation:
+#  https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md
+approvers:
+- fabriziopandini
+- neolit123
+reviewers:
```
Comment (yagonobre) [inline kinder/OWNERS]: Do you need to add you and @neolit123 as reviewer as well?

Comment (fabriziopandini) [reply kinder/OWNERS]: Good point. I will do this immediately after this one merge

**Comment by neolit123** [comment]
@timothysc mentioned that he is on PTO this week.
as per our previous agreement to have this tool as a replacement for kubeadm-playground i'm going to cancel the hold.

if we face complications and objections post merge we can discuss and amend.

/hold cancel

**Pull request merged**